tests/stratis_min.rs

```diff
@@ -214,6 +214,22 @@ fn test_stratis_min_pool_rename() {
     test_with_stratisd_min_sim(stratis_min_pool_rename);
 }
 
+#[test]
+// Test stopping a pool using an invalid name, that will fail with a
+// "Uuid error".
+fn stratis_min_pool_stop_invalid_name() {
```

Comment:
As written, the test is almost a parser test because the stratis-min code is interpreting the name as a UUID, trying to read it in, and failing, hence the UUID error. Since it never interacts with stratisd-min at all, you should use just the one part structure, like ```test_stratis_min_report_bad_subreport``` does.

Reply:
OK; I think I see what's going on.  I split the two tests; one stops a started pool, and the other is the parser test.